src/content/structured/styles/numbers-dates-time.mdx

```diff
@@ -7,7 +7,7 @@ date: "2024-04-19"
 
 title: "Numbers, dates and times"
```

Comment:
I think this should be 'time' not times as time would not be plural in this case?

Reply:
I do think times does make sense here. I can send some examples where times is used in this context :)